cms/test_utils/project/placeholderapp/views.py

```diff
@@ -24,7 +25,12 @@ def _base_detail(request, instance, template_name='detail.html',
         request.toolbar.set_object(instance)
     if template_string:
         template = Template(template_string)
-        return HttpResponse(template.render(RequestContext(request=request, dict_=context)))
+        if DJANGO_1_7:
+            return HttpResponse(template.render(RequestContext(request=request, dict_=context)))
+        else:
+            from django.template.context import make_context
+            context = make_context(context, request)
+            return HttpResponse(template.render(context))
```

Comment:
Can you do: `template.render(context, request)` It looks to me like that will take care of `make_context()` for you.
https://github.com/django/django/blob/11af73eaeb4371ded68460c1591abd6aefa57a90/django/template/backends/django.py#L63-L68